public/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>seaBay</title>
+</head>
+<body>
+  <div id="shoppingCart"></div>
+  <div id="searchBar"></div>
+  <div id="itemDetail"></div>
+  <div id="similarItems"></div>
+  <div id="reviews"></div>
```

Comment:
Change the name of these to match what other people are using to attach their components. I'm using `Cart` and @hadleyac is using `app`. We should formalize this.